### PR TITLE
Anya/1709 sign and certify save db

### PR DESCRIPTION
**Pull request by aroltsch**
connects #1778

### CODE REVIEW

**Comment by aroltsch** [comment]
Hey Nemo and Alex. Tests are failing. `Decision date` in the tests is not being set correctly due to the ID with spaces (ID should not contains spaces in HTML):
Here is the test snippet: page.execute_script("$('#Decision Date').val('02/01/2016')")
Here is the ID with spaces:
image
@nemodjuric @amprokop

**Comment by amprokop** [comment]
Anya is out so I'll take over this PR for her.

File: client/app/certification/SignAndCertify.jsx [hunk 1]
```diff
@@ -123,7 +144,7 @@ class UnconnectedSignAndCertify extends React.Component {
             required={true}
             onChange={onSignAndCertifyFormChange.bind(this, 'certifyingOfficialTitle')}/>
           <DateSelector
-            name="Decision Date:"
+            name="Date:"
```
Comment (amprokop) [inline client/app/certification/SignAndCertify.jsx]: this was wrong before, this is correct as per mocks

File: client/app/certification/actions/SignAndCertify.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    });
+
+  return {
+    type: Constants.CERTIFICATION_UPDATE_REQUEST,
+    payload: {
+      update
+    }
+  };
+};
+
```
Comment (sharonwarner) [inline client/app/certification/actions/SignAndCertify.js]: We should add unit tests for these.

**Pull request merged**